cli/foundation-models/nvidia-nemo/nemo_ptuning_summarization_with_textgen.ps1

```diff
@@ -0,0 +1,7 @@
+$ProgressPreference = 'SilentlyContinue'; Invoke-WebRequest -Uri https://aka.ms/installazurecliwindows -OutFile .\AzureCLI.msi; Start-Process msiexec.exe -Wait -ArgumentList '/I AzureCLI.msi /quiet'; Remove-Item .\AzureCLI.msi
+
+az extension add -n ml
+
+az login
+
+az ml job create -g <RESOURCE_GROUP> -w <WORKSPACE> --subscription <SUBSCRIPTION> -f nemo_pipeline_job.yml
```

Comment:
@vikasagrawal-ms Could we please get this working out of the box?

Reply:
Do we want to add, creation of compute as well in this.